README.md

```diff
@@ -1,753 +1,283 @@
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#oninit">onInit</a>** ‏‏‎ ‏‏‎ ‏‏‎           | Runs before anything else          |
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#onprebuild">onPreBuild</a>** ‏‏‎ ‏‏‎ ‏‏‎   | Before build commands are executed |
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#onbuild">onBuild</a>** ‏‏‎ ‏‏‎ ‏‏‎         | Build commands are executed        |
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#onpostbuild">onPostBuild</a>** ‏‏‎ ‏‏‎ ‏‏‎ | After Build commands are executed  |
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#onsuccess">onSuccess</a>** ‏‏‎ ‏‏‎ ‏‏‎     | Runs on build success              |
-| ⇩ ‏‏‎ ‏‏‎ ‏‏‎ **<a href="#onerror">onError</a>** ‏‏‎ ‏‏‎ ‏‏‎         | Runs on build error                |
-| 🎉 ‏‏‎ **<a href="#onend">onEnd</a>** ‏‏‎ ‏‏‎ ‏‏‎                    | Runs on build error or success     |
+For example, the `onPreBuild` event handler runs before your build command. Or the `onPostBuild` event handler runs
+after your site build has completed.
 
+### Available event handlers
+
+<!-- AUTO-GENERATED-CONTENT:START (LIFECYCLE_TABLE:noAnchors=true) -->
+| Event          | Description |
+|:------|:-------|
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onInit** ‏‏‎  ‏‏‎  ‏‏‎  | Runs before anything else |
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onPreBuild** ‏‏‎  ‏‏‎  ‏‏‎  | Before build commands are executed |
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onBuild** ‏‏‎  ‏‏‎  ‏‏‎  | Build commands are executed |
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onPostBuild** ‏‏‎  ‏‏‎  ‏‏‎  | After Build commands are executed |
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onSuccess** ‏‏‎  ‏‏‎  ‏‏‎  | Runs on build success |
+| ⇩ ‏‏‎  ‏‏‎  ‏‏‎ **onError** ‏‏‎  ‏‏‎  ‏‏‎  | Runs on build error |
+| 🎉 ‏‏‎ **onEnd** ‏‏‎  ‏‏‎  ‏‏‎  | Runs on build error or success |
 <!-- AUTO-GENERATED-CONTENT:END (LIFECYCLE_TABLE) -->
 
-The Lifecycle flows the events in order and executes and their `onPre` & `onPost` counterparts.
-
-`onPre` happens before a specific event.
-
-`onPost` happens after a specific event.
-
-```
-      ┌───────────────┬────────────────┬──────────────────┐
-      │     onPre     │     event      │      onPost      │
-      ├───────────────┼────────────────┼──────────────────┤
-      │               │                │                  │
-      │               │                │                  │
-...   │  onPreBuild   │    onBuild     │   onPostBuild    │   ...
-      │               │                │                  │
-      │               │                │                  │
-      └───────────────┤                ├──────────────────┘
-                      └────────────────┘
-
-      ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ━ ▶
-
-                        event flow
-```
-
-<!-- AUTO-GENERATED-CONTENT:START (LIFECYCLE_DOCS) -->
-
-### `onInit`
+### Anatomy of a plugin
 
-Runs before anything else
-
-<details>
-  <summary>Using <strong>onInit</strong> in a plugin</summary>
-  
-  <br/>
-
-Below is an example plugin using the `onInit` event handler
+Plugins are JavaScript objects like so:
```

Comment:
There is some additional documentation about `manifest.yml` that's been added yesterday: see [here](https://github.com/netlify/build/commit/f53dcb6928025ecbcedfa3e49f06423611604fce#diff-3d0480eda2fdf4a6ab5963612e72b8e2).